packages/material-ui-lab/src/TreeItem/TreeItem.js

```diff
@@ -263,33 +263,17 @@ const TreeItem = React.forwardRef(function TreeItem(props, ref) {
     ariaSelected = true;
   }
 
-  React.useEffect(() => {
-    if (nodeRef && treeId) {
-      const handleFocusIn = (event) => {
-        event.preventDefault();
```

Comment:
Nice, I feel like I’ve removed this so many times 😂 But I keep somehow adding it back.